pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -523,6 +527,7 @@ class Build {
                                             context.booleanParam(name: 'DYNAMIC_COMPILE', value: DYNAMIC_COMPILE),
                                             context.string(name: 'DOCKER_REGISTRY_URL', value: DOCKER_REGISTRY_URL),
                                             context.string(name: 'DOCKER_REGISTRY_URL_CREDENTIAL_ID', value: DOCKER_REGISTRY_URL_CREDENTIAL_ID),
+                                            context.string(name: 'BASE_DOCKER_REGISTRY_URL_CREDENTIAL_ID', value: BASE_DOCKER_REGISTRY_URL_CREDENTIAL_ID),
```

Comment:
`BASE_DOCKER_REGISTRY_CREDENTIAL_ID` is set above, but we are using `BASE_DOCKER_REGISTRY_URL_CREDENTIAL_ID` here (which is not being defined)